### DISCUSSION TITLE
Other Java Versions

### DISCUSSION THREAD

**Discussion by Veanuzz** [williamcorsel/hassio-addons#23] [Q&A]
Is it possible to run other server types, like paper? 
In the origin container was is possible, by configurating the type to paper. But i didn't found out, how it could work in the HASS add on.

**Comment by williamcorsel** [comment]
Reading the container docs the server type can be selected using the `TYPE` [environment variable](https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/). We could add this variable quite easily by adding it to the options in the `config.yaml`. However, the additional steps of making plugins or mod work might not be trivial in HAOS so I'm not sure if it would help you out much.

An alternative would be to use the [Portainer](https://github.com/alexbelgium/hassio-addons/tree/master/portainer) add-on and configure the container manually.

**Reply by Veanuzz** [reply]
Okay, i managed to configurate the type with the portainer add-on. 

But now i Unterstand the real problem. How do i manage to get access to the files from the server? I didnt understand the comment in the "issue" section. 

I tried to Change the path for the Container but could save. Any suggestions what i could do?

**Reply by williamcorsel** [reply]
If you're using Portainer you're not using this add-on anymore. You can refer to docker-minecraft-server [docs](https://docker-minecraft-server.readthedocs.io/en/latest/#using-docker-compose) for info on how to configure the container.
